pool/web/view/page/page_contact_completion.ml

```diff
@@ -21,16 +21,18 @@ let custom_field_to_input ?flash_fetcher language custom_field =
   match custom_field with
   | Public.Boolean (_, answer) ->
     answer
-    >|= (fun a -> a.Answer.value)
+    >>= (fun a -> a.Answer.value)
```

Comment:
this is used many times. How about adding a `value` function to the `Answer` module as well?